src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -292,25 +292,16 @@ export class RetrospectTaskModalComponent implements OnDestroy, AfterViewChecked
             {
                 headerName: 'Name',
                 colId: 'Name',
-                valueGetter: (params) => {
-                    return (params.data.FirstName + ' ' + params.data.LastName).trim();
-                },
+                valueGetter: (params) => (params.data.FirstName + ' ' + params.data.LastName).trim(),
```

Comment:
We can simplify this expression using `Object Destructuring` and `Template Literals`:
```js
valueGetter: ({data}) => `${data.FirstName} ${data.LastName}`.trim(),
```

References:
[Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
[Object Destructuring](https://stackoverflow.com/questions/17781472/how-to-get-a-subset-of-a-javascript-objects-properties)
[More on ES6 destructuring](https://gist.github.com/mikaelbr/9900818)

Reply:
I like the idea. @himanshu-garg-165 please see to it